frontier_assigner:
  start_time: 15
  info_radius: 3.0 # radius to consider laser readings from
  info_multiplier: 3.0 # a modifier for the info gains
  hysteresis_radius: 3.5 # at least as much as the laser scanner range
  hysteresis_gain: 2.0 # bigger than one (bias the robot to continue exploring the current region)
  filter_clear: 0.1
  filter_limit: 25